packages/admin/admin/src/error/inlineError/InlineError.tsx

```diff
@@ -0,0 +1,129 @@
+import { Error, Info, Warning } from "@comet/admin-icons";
+import { type ComponentsOverrides, type Theme, type Typography, useTheme } from "@mui/material";
+import { useThemeProps } from "@mui/material/styles";
+import { type FunctionComponent, type PropsWithChildren, type ReactNode } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { type ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
+import { ActionsContainer, Description, IconContainer, Root, Title } from "./InlineError.styles";
+
+export type InlineErrorClassKey = "root" | "iconContainer" | "title" | "description" | "actionsContainer";
+
+type InlineErrorVariant = "error" | "warning" | "info";
+
+type InlineErrorProps = ThemedComponentBaseProps<{
+    root: "div";
+    iconContainer: "div";
+    title: typeof Typography;
+    description: typeof Typography;
+    actionsContainer: "div";
+}> & {
+    /**
+     * A description message providing more details about the error.
+     */
+    description?: ReactNode;
+
+    /**
+     * A mapping of descriptions for each error variant.
+     */
+    descriptionMapping?: Record<InlineErrorVariant, ReactNode>;
+
+    /**
+     * Custom icon to be displayed next to the error message.
+     * If not provided, a default icon based on the variant will be used.
+     */
+    icon?: ReactNode;
+
+    /**
+     * A mapping of icons for each error variant.
+     */
+    iconMapping?: Record<InlineErrorVariant, ReactNode>;
+
+    /**
+     * Defines the type of error message displayed.
+     * Variants include "error", "warning", and "info".
+     *
+     * @defaultValue "error"
+     */
+    variant?: InlineErrorVariant;
```

Comment:
We could also name this `severity`, but in my opinion `variant` is more convenient

Reply:
I'd slightly prefer `severity` as this would leave open the option to add a `variant` in the future (maybe we want something like outlined or filled variants). 

But I'm fine with `variant` as well 😄

Reply:
MUI's Alert component has severity, varian and color: https://mui.com/material-ui/react-alert/. So maybe we should use severity as well?

Reply:
like we aligned, i renamed the component to `InlineAlert`